lms/static/js/groups/views/cohort_discussions_inline.js

```diff
@@ -17,10 +17,11 @@
                     },
 
                     render: function () {
-                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions');
+                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions'),
+                            inline_discussions = this.model.get('inline_discussions');
 
                         this.$('.cohort-inline-discussions-nav').html(this.template({
-                            inlineDiscussionTopics: this.getInlineDiscussionsHtml(this.model.get('inline_discussions')),
+                            inlineDiscussionTopicsHtml: this.getInlineDiscussionsHtml(inline_discussions),
```

Comment:
That's correct. It would escape HTML that you don't want escaped.

> On Mar 25, 2016, at 11:51 AM, Christina Roberts <notifications@github.com> wrote:
> 
> In lms/static/js/groups/views/cohort_discussions_inline.js:
> 
> >  
> >                          this.$('.cohort-inline-discussions-nav').html(this.template({
> > -                            inlineDiscussionTopics: this.getInlineDiscussionsHtml(this.model.get('inline_discussions')),
> > +                            inlineDiscussionTopicsHtml: this.getInlineDiscussionsHtml(inline_discussions),
> @robrap To clarify, I can't use HtmlUtils.ensureHtml unless I first wrap the HTML as an HTML snippet, correct?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>